src/main/java/com/coinbase/prime/model/orders/AcceptQuoteRequest.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2025-present Coinbase Global, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.coinbase.prime.model.orders;
+
+import com.coinbase.core.errors.CoinbaseClientException;
+import com.coinbase.prime.model.enums.OrderSide;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.coinbase.core.utils.Utils.isNullOrEmpty;
+
+public class AcceptQuoteRequest {
+    @JsonProperty(required = true, value = "portfolio_id")
+    @JsonIgnore
```

Comment:
ignore and required?

Reply:
ignore when sending request, required for putting in url. this is correct